src/recon_test/recontest.cxx

```diff
@@ -24,7 +24,6 @@
 #include "stir/Succeeded.h"
```

Comment:
actually. changes to this file shouldn't really be in a listmode PR. Depending on how long we take with the current PR, maybe you want to create a separate one (but I don't insist!)

Reply:
you're right but ... pls let's leave it here

Reply:
I'll probably squash-merge this PR (unless you disagree). In that case, it will seem rather weird to change recontest in a listmode commit. I'd recommend a new PR for that file, and then a merge with master again here. should take max 15 mins.

Reply:
A squash is absolutely alright